src/SWP/Bundle/CoreBundle/Form/Type/ContentListPositionType.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2019 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2019 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class ContentListPositionType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add('id', IntegerType::class);
```

Comment:
IMO you should write and use here a custom type, e.g. `ContentListSelectorType` which will use data transformer to transform content list id to Content List object and vice versa. Because at the moment you can type whatever id you want here and no validation will be triggered.

Reply:
But for what when in database we keep it as serialized array? We don't need at this moment those content lists objects at all.

Reply:
for validation purposes, to not enter not existing content list id and do not store it in the database. Why would you want to store not existing content list id?

Reply:
or just add a custom validator to check if the list with provided id exists if these content lists objects are not needed